assets/js/vue/components/research-group/OverviewTab.vue

```diff
@@ -13,7 +13,7 @@
                 {{ overview.fundingCycle.fundingOrganization.name }}
             </a>
         </p>
-        <h6 class="font-weight-bold">Funding Cycle</h6>
+        <h6 class="font-weight-bold">{{ getFcLabel() }}</h6>
```

Comment:
I don't think GRIIDC users can get to this page, even if we want a conditional rendering i think it could all go into a JS util file